netuitive/conf/netuitive-agent.conf

```diff
@@ -139,10 +140,10 @@ include_cpu_pct = True
 enabled = True
 # exclude everything that begins /boot or /mnt
 exclude_filters = ^/boot, ^/mnt
-simple = True
```

Comment:
yes, that'd be case based on diamond config load. since it's already True as default, we no longer to define it twice here with the same value.